server/store/db.js

```diff
@@ -0,0 +1,54 @@
+var mongoose = require('mongoose'),
+    config = require('../config/config.json').mongo;
+
+var connectWithRetry = function() {
+  return mongoose.connect(
+    config.server + config.databaseName,
+    config.connection_options,
```

Comment:
let's use camelCase everywhere we can to keep code naming convention consistent `connectionOptions` instead of `connection_options`